### PR TITLE
Initial Linux Test Support

### PR DESCRIPTION
**Pull request by slide**
Fixes some issues with getting tests running on Linux

### CODE REVIEW

File: Languages/IronPython/IronPython.Modules/nt.cs [hunk 1]
```diff
@@ -812,9 +812,9 @@ public static class PythonNT {
             process.StartInfo.Verb = operation;
 #endif
             try {
-
                 process.Start();
             } catch (Exception e) {
+				Console.Error.WriteLine(e.ToString());
```
Comment (slozier) [inline Languages/IronPython/IronPython.Modules/nt.cs]: I'm guessing this is for debugging?

File: Languages/IronPython/Tests/test_cliclass.py [hunk 2]
```diff
@@ -1436,6 +1436,7 @@ def test_clr_dir():
     Assert('IndexOf' in clr.DirClr('abc'))
 
 def test_array_contains():
+    print '%s' % repr(System.Array[str].__dict__)
```
Comment (slozier) [inline Languages/IronPython/Tests/test_cliclass.py]: for debugging?

File: Languages/IronPython/Tests/test_execfile.py [hunk 3]
```diff
@@ -14,16 +14,19 @@
...
 
+import os
+
 
 @skip("win32")
 def test_sanity():
     root = testpath.public_testdir
-    execfile(root + "/Inc/toexec.py")
-    execfile(root + "/Inc/toexec.py")
+    execfile(path_combine(root, "Inc", "toexec.py"))
```
Comment (slozier) [inline Languages/IronPython/Tests/test_execfile.py:26]: Could we use `os.path.join` instead?

**Review by slozier — COMMENTED** [review]
LGTM. I should have tried doing a diff without the nt to os changes.

File: Languages/IronPython/IronPython/Lib/iptest/assert_util.py [hunk 4]
```diff
@@ -382,6 +385,8 @@ def multiple_execute_test(self):
 		return get_num_iterations() > 1
     def stdlib_test(self):
         return is_stdlib()
+    def posix_test(self):
+		return is_posix
```
Comment (slozier) [inline Languages/IronPython/IronPython/Lib/iptest/assert_util.py]: nit: missing a space here

File: Languages/IronPython/IronPython.Modules/nt.cs (hunk 1 above)
Comment (slide) [reply Languages/IronPython/IronPython.Modules/nt.cs]: Yes, committed some stuff by mistake.

File: Languages/IronPython/Tests/test_cliclass.py (hunk 2 above)
Comment (slide) [reply Languages/IronPython/Tests/test_cliclass.py]: Yes, will clean-up

File: Languages/IronPython/Tests/test_execfile.py (hunk 3 above)
Comment (slide) [reply Languages/IronPython/Tests/test_execfile.py:26]: I wondered this too...not sure why they didn't just import os.

**Pull request merged**